sunpy/util/metadata.py

```diff
@@ -9,7 +9,8 @@
 
 class MetaDict(OrderedDict):
     """
-    A class to hold metadata associated with a `sunpy.map.Map
+    A class to hold metadata associated with a `sunpy.map.Map.
+
     <sunpy.map.map_factory.MapFactory.__call__>` derivative.
```

Comment:
```suggestion
    A class to hold metadata associated with a
    `sunpy.map.Map <sunpy.map.map_factory.MapFactory>` derivative.
```
Why did it think the space within a link was a place to end a sentence?